doc/source/whatsnew/v1.4.0.rst

```diff
@@ -636,7 +636,7 @@ Conversion
 - Bug in :class:`IntegerDtype` not allowing coercion from string dtype (:issue:`25472`)
 - Bug in :func:`to_datetime` with ``arg:xr.DataArray`` and ``unit="ns"`` specified raises TypeError (:issue:`44053`)
 - Bug in :meth:`DataFrame.convert_dtypes` not returning the correct type when a subclass does not overload :meth:`_constructor_sliced` (:issue:`43201`)
--
+- Bug in :meth:`DataFrame.astype` not propagating ``attrs`` from the original :class:`DataFrame` (:issue:`44414`)
```

Comment:
can you move this next to the other notes about attr changes

Reply:
Yes I can, I only found mentions of Attributes being changed within I/O and sparse section. Should I move it to one of those? Thank you in advance for the help